k9mail-library/src/test/java/com/fsck/k9/mail/internet/MessageExtractorTest.java

```diff
@@ -46,9 +40,9 @@ public void getTextFromPart_withNoBody_shouldReturnNull() throws Exception {
 
     @Test
     public void getTextFromPart_withTextBody_shouldReturnText() throws Exception {
-        withMimeType("text/plain");
-        withContentType("UTF-8");
-        withBody(createTextBody("Sample text body"));
```

Comment:
Just using the parts without mocks is so simple these methods ended up as just a bunch of delegated setters, I felt like there was no point in adding an abstraction...